site/_docs/reference.md

```diff
@@ -2620,6 +2622,8 @@ semantics.
 | b | DATETIME(date, time)                           | Returns a datetime object, given a date and a time.
 | b | DATETIME(timestamp)                            | Converts a timestamp object to a datetime, assuming UTC.
 | b | DATETIME(timestamp, timezone)                  | Converts a timestamp object to a datetime, in a given time zone.
+| b | DATETIME_ADD(timestamp, interval)              | Synonym for TIMESTAMP_ADD.
```

Comment:
remove '.'

Not really a synonym. The implementation happens to allow both TIMESTAMP and TIMESTAMP WITH LOCAL TIME ZONE, and therefore DATETIME_ADD shares an implementation with TIMESTAMP_ADD. But the user doesn't have to know that.